Lib/gflanguages/languages_public.proto

```diff
@@ -48,8 +48,9 @@ message ExemplarCharsProto {
   optional string numerals = 4;
   optional string punctuation = 5;
   optional string index = 6;
+  optional string removable = 7;  // Base characters which can be ignored when determining language support
```

Comment:
nit: Move comment above field. I would also suggest a different name.

```
// Base characters which can be ignored when determining language support
optional string not_required = 7;
```

Alternative name ideas: not_required, optional_base, base_optional. I think that `removable` sounds a bit misleading, as if the character might be subset out.

Reply:
> nit: Move comment above field. 

All other field comments are inline:

```
message ScriptProto {
  optional string id = 1;  // Script codes defined by CLDR
  optional string name = 2;

  // Next = 3;
}

message LanguageProto {
  optional string id = 1;  // Either ${lang} or ${lang}_${script}
  optional string language = 2;  // BCP 47
  optional string script = 3;
  optional string name = 4;
  optional string preferred_name = 5;
  optional string autonym = 6;  // Name of language as written in that language
  optional int32 population = 7;
```

Reply:
Touché! Thanks, Simon. Used to seeing it another way.